app/components/quiz/Quiz.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import AppStore from 'store/appStore';
+import QuizQuestion from 'components/quiz/QuizQuestion.js';
+
+import '../../styles/quiz.scss';
+
+export default React.createClass({
+  displayName: 'Quiz',
+
+  contextTypes: {
+    router: React.PropTypes.func
+  },
+
+  propTypes: {
+    questions: React.PropTypes.array.isRequired
+  },
+
+  getDefaultProps: function() {
+    return {
```

Comment:
ideally i'd like to see this worked into an action so that we can be prepared to consume actual requests when the time comes. or at least a #TODO comment for the time being.